bigchaindb/web/views/parameters.py

```diff
@@ -0,0 +1,30 @@
+import re
+
+
+def valid_txid(txid):
+    if re.match('^[a-fA-F0-9]{64}$', txid):
+        return txid.lower()
+    raise ValueError("Invalid hash")
+
+
+def valid_bool(val):
+    if val == 'true':
+        return True
+    if val == 'false':
+        return False
+    raise ValueError('Boolean value must be "true" or "false" (lowercase)')
```

Comment:
Using `type=bool` in [`reqparse.RequestParser.add_argument(*args, **kwargs)`](http://flask-restful.readthedocs.io/en/0.3.5/api.html#reqparse.RequestParser.add_argument) may be enough?